base-notebook/Dockerfile

```diff
@@ -78,13 +78,15 @@ RUN pip --no-cache-dir install 'jupyterhub==0.5'
 
 USER root
 
-# Configure container startup as root
 EXPOSE 8888
 WORKDIR /home/$NB_USER/work
+
+# Configure container startup
 ENTRYPOINT ["tini", "--"]
-CMD ["start-notebook.sh"]
+CMD ["start-jupyter.sh", "notebook"]
```

Comment:
Test runner was not using exit code 1 when every test retry failed. The test was indeed failing with the bad start command, but the travis job was not. Fixed both the start command and the make test target.